demos/atb-agg.c

```diff
@@ -423,6 +423,18 @@ void TW_CALL get_tertiary( void *value, void *data )
 // ------------------------------------------------------------------- init ---
 void init( GLFWwindow* window )
 {
+    int is_scaled;
+    {
+        int pixWidth, pixHeight, winWidth, winHeight;
+        glfwGetFramebufferSize( window, &pixWidth, &pixHeight );
+        glfwGetWindowSize( window, &winWidth, &winHeight );
+        is_scaled = pixWidth > winWidth || pixHeight > winHeight;
```

Comment:
If `is_scaled` is defined as `bool`, I get the following compilation error:

    demos/atb-agg.c:433:5: error: use of undeclared identifier 'bool'
        bool is_scaled;
        ^

I'm used to using `bool` in C++, but afaik it isn't available in plain C, which I haven't used in quite a while, so I'm not sure what to use.. What do you suggest?